frontend/webapp/configs/logging.yaml

```diff
@@ -36,7 +36,7 @@ handlers:
     #     address: [localhost, /dev/log]
 
 loggers:
-    webapp_logger:
+    artemis_logger:
```

Comment:
reminder: people will need to change this in their local_configs, otherwise things will break on the frontend